lib/rubycritic/cli/application.rb

```diff
@@ -14,12 +14,29 @@ def initialize(argv)
 
       def execute
         parsed_options = @options.parse
-        ::Rubycritic::CommandFactory.create(parsed_options).execute
+        result = ::Rubycritic::CommandFactory.create(parsed_options).execute
```

Comment:
And since we're on it, let's drop the scope operator as well. Just

>> Rubycritic::CommandFactory.create...

is fine.